src/sentry/models/project.py

```diff
@@ -98,6 +98,7 @@ class Project(Model):
         'pk',
         'slug',
     ])
+    platform = models.CharField(max_length=64, null=True)
```

Comment:
this may become an explicit config, and its one per project. We discussed this internally already and decided to go this way.

You can imagine a day soon when you select JS, and you get an icon associating your project with JS.